lib/ansible/parsing/vault/__init__.py

```diff
@@ -75,7 +74,7 @@
 from ansible.errors import AnsibleError
 from ansible import constants as C
 from ansible.module_utils.six import PY3, binary_type
-from ansible.module_utils.six.moves import zip
+from ansible.module_utils.six.moves import zip as _zip
```

Comment:
I don't mind it myself because the 'zip' I have imported is (sometimes) not the builtin 'zip', so the different name makes sense. Also generally dislike shadowing so I'd like to keep it for the general case.

But granted, 'six' exists to abstract those details...

pylint has a built in 'redefining-builtins-modules' option that by default includes 'six.moves,future.builtins' so should be able to add 'ansible.module_utils.six.moves' to the exceptions.